### ISSUE TITLE
pulling the same Airport trees numerous times in a run...

### ISSUE THREAD

**Issue by wkitty42** [open744/terramaster#12]
here's an example... note that it pulled U/I/9 twice and tried to pull U/I/4 twice...

>sync e100n50/e106n51... Terrain/e100n50/e106n51/69 items (5974512 bytes)... updated to r56468.
Airports/U/I/9/5 items (1532 bytes)... updated to r56468.
Airports/U/I/6/12 items (3685 bytes)... updated to r56468.
Airports/U/I/4 not found
updated to r56468.
Airports/U/I/3/3 items (761 bytes)... updated to r56468.
Objects/e100n50/e106n51 not found
updated to r56468.
[...]
sync e100n50/e107n51... Terrain/e100n50/e107n51/70 items (6085813 bytes)... updated to r56468.
Airports/U/I/U/11 items (7331 bytes)... updated to r56468.
Airports/U/I/9/5 items (1532 bytes)... updated to r56468.
Airports/U/I/7/4 items (974 bytes)... updated to r56468.
Airports/U/I/4 not found
updated to r56468.

this is not the only example... if i can figure out how to have all the console output go to a log file, i can show more... (i haven't tried using tee yet but i will :smile_cat:)

**Comment by Portree-Kid** [comment]
Can't set it to accepted but I'll fix it. Currently also happening in the new HTTP Based code.

**Comment by wkitty42** [comment]
yeah, i know why it is doing it but not sure why the existing code can't tell that it has just been updated... i suspect that reading the cache first will be able to tell when the last time it was updated and avoid repeated downloads like these...

**Comment by Portree-Kid** [comment]
Yes of course. I'll fix it.

**Comment by Portree-Kid** [comment]
Fixed by my pull request. There is also a release in my fork.
